README.md

```diff
@@ -1,12 +1,5 @@
 # AT Driver API Explainer
 
-**at-driver** &middot; [aria-at-automation-harness](https://github.com/w3c/aria-at-automation-harness) &middot; [aria-at-automation-driver](https://github.com/w3c/aria-at-automation-driver) &middot; [aria-at-automation-results-viewer](https://github.com/w3c/aria-at-automation-results-viewer)
-
-## Authors
-
-* Mike Pennisi
-* Simon Pieters
-
```

Comment:
Why was this removed?

Reply:
The proposal itself includes attribution.